localdashboard_cumulus_std.bash

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# SIT shared dashboard
+export APIROOT=https://d84ilrgn1b7e1.cloudfront.net/
```

Comment:
This config value could change.   This may also be a security/obscurity concern.